image/templates/helm/stackrox-central/internal/defaults.yaml.htpl

```diff
@@ -314,6 +314,10 @@ defaults:
         tag: [< required "" .ScannerV4ImageTag >]
     exposeMonitoring: false
 
+  configAsCode:
+    enabled: true
```

Comment:
enabled should technically be false till we turn on feature flag by default. If we for any reason make the decision to not ship with feature flag enabled by default this release, we should not be deploying config controller?

Reply:
Should we remove the feature flag now that we can disable policy-as-code via helm chart values?  The main thing we lose is protection of master from silly Helm chart template generation goofs (which we recently had!).

Reply:
@clickboo I know we talked about this last week, and I think you ended up being _okay_ about removing the feature flag since we will still have a way to disable the deployment through the helm chart.  Is that correct?